lib/storage/src/content_manager/conversions.rs

```diff
@@ -68,6 +68,7 @@ impl TryFrom<api::grpc::qdrant::CreateCollection> for CollectionMetaOperations {
                     .map(sharding_method_from_proto)
                     .transpose()?,
                 strict_mode_config: value.strict_mode_config.map(strict_mode_from_api),
+                uuid: None,
```

Comment:
This is conversion from gRPC `CreateCollection` into Qdrant `CreateCollection`.

IMO, the user should not be allowed to explicitly specify collection UUID (it should be randomly generated), so it makes sense to **exclude** UUID from gRPC.

As far as I can tell gRPC `CreateCollection` is only used as part of **public** `create_collection` gRPC API, but not in internal gRPC API. When Raft propagates `CreateCollection` message to other nodes, it does not use gRPC directly, but serialize *Qdrant* `CreateCollection` structure (which contains UUID) to CBOR.